src/Signature/SignatureV4.php

```diff
@@ -285,22 +287,22 @@ private function getCanonicalizedQuery(array $query)
         return substr($qs, 0, -1);
     }
 
-    private function convertToTimestamp($dateValue)
+    private function convertToTimestamp($dateValue, $relativeTimeBase = null)
     {
         if ($dateValue instanceof \DateTime) {
             $timestamp = $dateValue->getTimestamp();
         } elseif (!is_numeric($dateValue)) {
-            $timestamp = strtotime($dateValue);
+            $timestamp = strtotime($dateValue, $relativeTimeBase === null ? time() : $relativeTimeBase );
```

Comment:
Please break this into multiple lines. We try to keep all line length to be of less than 80 characters in the PHP SDK.